packages/cli/src/daemon/DaemonClient.ts

```diff
@@ -206,13 +189,13 @@ export class DaemonClient {
                 this.shutdown();
             }
         } else {
-            if (stdinData != null) {
-                await this.writeToStdin(stdinData, jsonData.stdinLength);
-            }
-
             Imperative.commandLine = jsonData.argv.join(" ");
             Imperative.api.appLogger.trace(`daemon input command: ${Imperative.commandLine}`);
-            Imperative.parse(jsonData.argv, { stream: this.mClient, daemonResponse: jsonData });
+            const context: IDaemonContext = { stream: this.mClient, response: jsonData };
```

Comment:
That's right, since the interface is now called `IDaemonContext` instead of `IYargsContext`, including "daemon" in the property name seemed redundant 😛